frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionCreateYaml.tsx

```diff
@@ -31,7 +31,7 @@ const CreateNetAttachDefYAMLConnected = connectToPlural(
 
     const netAttachDefTemplatePath = (o: K8sResourceKind) =>
       resourcePathFromModel(
-        { ...NetworkAttachmentDefinitionModel, plural: 'networkattachmentdefinitions' },
+        { ...NetworkAttachmentDefinitionModel, plural: 'network-attachment-definitions' },
```

Comment:
Can we use just `NetworkAttachmentDefinitionModel` ?
```
resourcePathFromModel(NetworkAttachmentDefinitionModel, ....
```

Reply:
Fixed in https://github.com/openshift/console/pull/3183/